autoload/airline/extensions/searchcount.vim

```diff
@@ -18,7 +18,7 @@ function! airline#extensions#searchcount#apply(...) abort
 endfunction
 
 function! airline#extensions#searchcount#status() abort
-  let result = searchcount(#{recompute: 1})
+  let result = searchcount(#{recompute: 1, maxcount: -1})
```

Comment:
I am not sure about this. This might potentially slow down Vim considerably, if it needs to get all counts. Did you try this with a long document and a pattern that matches a lot?

Reply:
@chrisbra 
Thanks for your code review.
I have tested this extension in a situation where there are over 3000 search patterns and it works fine.

Reply:
so let's await complaints :)

Reply:
This is slowing up for me when I opened a not so large gz file (56MB). How do I configure the limit?

Reply:
@rulero

Does adding this configuration to vimrc fix the problem?

```vim
let g:airline#extensions#searchcount#enabled = 0
```